crates/astria-core/src/sequencer/v1alpha1/block.rs

```diff
@@ -565,3 +568,77 @@ where
     };
     merkle::Tree::from_leaves(iter.into_iter().map(|item| Sha256::digest(&item)))
 }
+
+/// [`Deposit`] represents a deposit from the sequencer to a rollup.
+///
+/// A [`Deposit`] is constructed whenever a [`BridgeLockAction`] is executed
+/// and stored as part of the block's events.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Deposit {
```

Comment:
This crate tries to keep most fields private by default while this type doesn't. I grepped for a few fields but couldn't quickly find direct field accesses.

Can we keep these fields private?